test/metabase/logger_test.clj

```diff
@@ -56,4 +57,21 @@
     (is (not (some (fn [{message :msg, :as entry}]
                      (when (str/includes? (str message) "THIS SHOULD NOT SHOW UP")
                        entry))
-                   (mb.logger/messages))))))
+                   (logger/messages))))))
+
+(deftest fork-logs-test
+  (testing "logger/for-ns works properly"
+    (let [f (io/file (System/getProperty "java.io.tmpdir") (mt/random-name))]
+      (try
+        (with-open [_ (logger/for-ns 'metabase.logger-test f {:additive false})]
+          (log/info "just a test"))
+        (is (=? #".*just a test\n"
+                (slurp f)))
```

Comment:
can you add tests to make sure after we close the outputstream, the logger doesn't keep appending to it?